gradio/components/interpretation.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         Component.__init__(
             self, visible=visible, elem_id=elem_id, elem_classes=elem_classes, **kwargs
         )
-        self.component = component
```

Comment:
Why did you make this change? It broke interpretation, test e.g. with: `demo/gender_sentence_default_interpretation/run.py`

Reply:
oh hmm well interpretation is broken in main, try titanic_survival demo. I made this fix for titanic_survival but I guess I broke gender_sentence_default_interpretation instead. Will revert this change, and interpretation can be properly fixed in another PR.